src/vmecpp/__init__.py

```diff
@@ -830,7 +830,31 @@ def _to_cpp_wout(self) -> _vmecpp.WOutFileContents:
 
         return cpp_wout
 
-    # TODO(eguiraud): implement from_wout_file
+    @staticmethod
+    def from_wout_file(wout_filename: str | Path) -> VmecWOut:
+        """Load wout contents in NetCDF format.
+
+        This is the format used by Fortran VMEC implementations and the one expected by
+        SIMSOPT.
+        """
+        with netCDF4.Dataset(wout_filename, "r") as fnc:
+            fnc.set_auto_mask(False)
+            attrs = {}
+            for key in fnc.variables:
+                if key.endswith("__logical__"):
+                    attrs[key[:-11]] = fnc[key][()] != 0
```

Comment:
let's make the intent clearer:

```suggestion
                    attrs[key.removesuffix("__logical__")] = fnc[key][()] != 0
```